Android/Fundamentals/intro-to-android/hello-android.md

```diff
@@ -0,0 +1,53 @@
+# hello-android
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+We are going to create a simple Hello Android app using the following directions below:
+Open Android Studio and on the Welcome menu, click on Start a New Android Studio Project. A new project window opens and the following information will appear to include: 
+
+*	Application Name. This will appear to users. Example name it First App. 
+*	Company Domain. This is the name of your company. This is used to generate the package name below the field  
+  and a package namespace for your app. 
+*	Project Location. This is the location in which you want to place your project in. Click Next. 
+* Leave Phone and Table checked and leave the Minimum SDK at the default. Click Next. 
+* Select Empty Activity selected and click Next.
+*	Leave all the details for the activity in their default state. Click Finish.
+*	Android Studio may take at least a minute or two to set up. 
```

Comment:
Would be great if we explained why it takes longer the first time around. For example : because Gradle is getting built for the first time and it's pulling in all the dependencies that the project requires